src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -130,18 +196,16 @@ def main():
 
     # check if input file exists
     if not gs.find_file(input)["file"]:
-        gs.fatal(_("Raster map <%s> not found") % input)
+        gs.fatal(_("Raster map {} not found".format(input)))
```

Comment:
The previous string translates well. Was there a conflict solved manually when creating the new PR? I solved all of the string translation issues in the default branch a couple months ago.

Reply:
Just that it was once mentioned that using the .format form was preferred, but perhaps that has changed?

Reply:
The preference isn't the problem here. The problem is that the function named `_`, `_()`, takes a string, searches for the string in the translated strings, and returns the translated string. When translating, we leave the placeholders as they can move in different languages.
Here, the string formatting was done, replacing the placeholders, and searches for that string (without any placeholders): so no match.
What should be done, is to get the translated string first, then format it replacing the placeholders.

Then, a second consideration is that collecting strings to translate isn't currently up to date, so changing strings uselessly will just have misses as they wouldn't be translated. Having multiple places using the same, existing, standard strings helps the translation work.

Reply:
I am not sure if I follow completely, and I am more than happy to revert to what it was originally. However, as I used this format in many other places, I want to make sure I understand. Is (one of) the issue(s) that I misplaced one of the brackets, and should it be?:

```
gs.fatal(_("Raster map {} not found").format(input))
```

Reply:
Then, if changing from % to .format is the only change, without other improvements or changes, then it is useless to degrade translation work by adding a new untranslated string (that isn't collected for now, it is something I'd like to work on in the dev summit in may).
Changes to use an existing string is fine though.